ssl/internal.h

```diff
@@ -2425,12 +2425,14 @@ bool tls12_check_peer_sigalg(const SSL_HANDSHAKE *hs, uint8_t *out_alert,
 // From RFC 4492, used in encoding the curve type in ECParameters
 #define NAMED_CURVE_TYPE 3
 
-struct CERT {
-  static constexpr bool kAllowUniquePtr = true;
-
-  explicit CERT(const SSL_X509_METHOD *x509_method);
-  ~CERT();
-
+// SSL_PKEY_* denote certificate types. These represent an index within
+// |CERT->cert_privatekeys|.
+#define SSL_PKEY_RSA 0
+#define SSL_PKEY_ECC 1
+#define SSL_PKEY_ED25519 2
+#define SSL_PKEY_NUM 3
```

Comment:
Should this be called `SSL_PKEY_COUNT` or `SSL_PKEY_SIZE`? It's only used to indicate the size of the array.